src/components/templates/ExternalRoom/ExternalRoom.scss

```diff
@@ -38,4 +40,11 @@
     font-style: normal;
     font-weight: $font-weight--500;
   }
+
+  &__background {
```

Comment:
this should go to ContainerWithBackground, i believe.

Reply:
fixed b396ea2